ui/lib/pki/addon/components/page/pki-configuration-edit.hbs

```diff
@@ -126,24 +126,22 @@
     <div class="field is-grouped box is-fullwidth is-bottomless">
       <div class="control">
```

Comment:
This can be replaced with `<Hds::ButtonSet>`

Reply:
I was curious about this. It's a conditional so it could be only one button displayed, but is the idea to remove the class=control div?